rpc/jsonrpc/client/http_uri_client.go

```diff
@@ -54,6 +55,7 @@ func NewURI(remote string) (*URIClient, error) {
 func (c *URIClient) Call(ctx context.Context, method string,
 	params map[string]interface{}, result interface{}) (interface{}, error) {
 
+	<-time.After(1 * time.Second)
```

Comment:
This seems like a hack to me.

Reply:
See comment below.  The goal is not to merge this particular line.